test/unit/walk-manifest.spec.js

```diff
@@ -688,37 +725,81 @@ describe('walk-manifest', function() {
           assert.equal(count.m4a, 5, 'm4a count as expected');
         });
     });
-  });
```

Comment:
This needs to go below all of the tests, otherwise they are not all part of the same module